buildSrc/genVSproperties.bat

```diff
@@ -21,79 +21,69 @@ REM Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
 REM or visit www.oracle.com if you need additional information or have any
 REM questions.
 
-REM Windows bat file that runs vcvars32.bat for Visual Studio 2003
-REM   and echos out a property file with the values of the environment
-REM   variables we want, e.g. PATH, INCLUDE, LIB, and LIBPATH.
+setlocal ENABLEDELAYEDEXPANSION
+
+REM Windows bat file that runs vcvars64.bat for Visual Studio
+REM and echos out a property file with the values of the environment
```

Comment:
nit : I think it is usually spelt echoes.

Reply:
This was preexisting, but since I touched this line (to remove white space), I'll fix it.